hazelcast/src/main/java/com/hazelcast/sql/impl/exec/root/BlockingRootResultConsumer.java

```diff
@@ -33,6 +33,9 @@
     /** Iterator over produced rows. */
     private final InternalIterator iterator = new InternalIterator();
 
+    /** Enables {@link ResultIterator#RETRY} result from {@link ResultIterator#hasNextImmediately()} */
+    private final boolean useMinimumLatency;
```

Comment:
A better name for this would be something like `waitForFullBatch`.

Reply:
fixed